modules/blockfrost.nix

```
@@ -0,0 +1,82 @@
+{
+  config,
+  lib,
+  ...
+}: let
+  cfg = config.cardano.blockfrost;
+  dbsync-cfg = config.services.cardano-db-sync;
```

```suggestion
  dbsync-cfg = config.services.cardano-db-sync or null;
```
we have to check the module exists